### ISSUE TITLE
Changing default configuration to Normal cast.

### ISSUE THREAD

**Issue by joshjayk** [loopuleasa/Dota2-TheCore-Config-Engine#99]
As far as I know, there is no way to currently set the default to Normal cast. This means that every time I open the Dota client, my very first game I have to hit the F7 button. Related to this, the F7 button does not work initially. I hit it and then the prompt says "All spells and abilities set to Quick cast", even though it was quick cast to begin with. Then, I have to hit F7 again to actually make it normal cast.

Tl;dr: I have to hit F7 twice in my very first game in order to change from quick to normal cast. Also, the prompt is prompted to "say_team" not the coach.

**Comment by loopuleasa** [comment]
Yes, this is an initialization bug and it will probably be resolved in the future.

As for switching up the default mode at launch, that can be fixed with code changes of the toggle functions.

**Comment by Sembrani** [comment]
Try adding "quick_cast_toggled_off" - no quotes, at the end of your autoexec.cfg file, and let me know if that works
